tests/imagebasedupgrade/tests/happy-path-upgrade.go

```diff
@@ -0,0 +1,58 @@
+package ran_du_system_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/lca"
+	"github.com/openshift-kni/eco-goinfra/pkg/polarion"
+	"github.com/openshift-kni/eco-gosystem/tests/imagebasedupgrade/internal/imagebasedupgradeinittools"
+	"github.com/openshift-kni/eco-gosystem/tests/imagebasedupgrade/internal/imagebasedupgradeparams"
+)
+
+// IbuCr is a dedicated var to use and act on it.
+var IbuCr = lca.NewImageBasedUpgradeBuilder(
+	imagebasedupgradeinittools.TargetSNOAPIClient, imagebasedupgradeparams.ImagebasedupgradeCrName)
+
+var _ = Describe(
+	"HappyPathUpgrade",
+	Ordered,
+	ContinueOnFailure,
+	Label("HappyPathUpgrade"), func() {
+		BeforeAll(func() {
+			By("Generating seed image", func() {
+
+			})
+		})
+
+		AfterAll(func() {
+			// Rollback to pre-upgrade state and reset PolicyGenTemplate.
+		})
+
+		It("End to end upgrade happy path", polarion.ID("68954"), Label("HappyPathUpgrade"), func() {
+			By("Installing and Configuring OADP operator via GitOps ZTP in Target SNO", func() {
```

Comment:
I think we can install OADP operator as part of the target SNO initial installation so we can skip this step in the test.

Reply:
Ok, done.